PyStemmusScope/forcing_io.py

```diff
@@ -189,4 +199,36 @@ def prepare_forcing(config):
 
     # Write the remaining variables (without time dependency) to the matlab v7.3
     #  file 'forcing_globals.mat'
-    prepare_global_variables(data, input_path, config)
+    prepare_global_variables(data, input_path)
+
+
+def _slice_forcing_file(ds_forcing, start_time, end_time):
+    """Get the subset of forcing file based on time range in config
+
+    Also check if the desired time range is covered by forcing file.
+
+    Args:
+        ds_forcing (xr.Dataset): Dataset of forcing file.
+        start_time (str): Start of time range in ISO format string e.g. 'YYYY-MM-DDTHH:MM:SS'. If "NA", start time will be the
+            first timestamp of the forcing input data.
+        end_time (str): End of time range in ISO format string e.g. 'YYYY-MM-DDTHH:MM:SS'. If "NA", end time will be the
+            last timestamp of the forcing input data.
```

Comment:
```suggestion
        start_time (str): Start of time range in ISO format string e.g. 'YYYY-MM-DDTHH:MM:SS'. 
            If "NA", start time will be the first timestamp of the forcing input data.
        end_time (str): End of time range in ISO format string e.g. 'YYYY-MM-DDTHH:MM:SS'. 
            If "NA", end time will be the last timestamp of the forcing input data.
```

Reply:
My suggestion will fix the linter error "Line too long". It seems it is not picked by `prospector` in the github action, but `pylint` is complaining about it :thinking:

Reply:
OK! It might have to do with your local settings then, because I don't see line-too-long errors on docstrings. But I'll commit your suggestions.